src/test/regress/sql/failure_online_move_shard_placement.sql

```diff
@@ -131,9 +131,14 @@ SELECT master_move_shard_placement(101, 'localhost', :worker_1_port, 'localhost'
 
 SELECT citus.mitmproxy('conn.allow()');
 -- first, manually drop the subscsription object. But the record for it will remain on pg_dist_cleanup
-SELECT run_command_on_workers($$DROP SUBSCRIPTION IF EXISTS citus_shard_move_subscription_10_15$$);
+-- we expect the drop query will succeed on only one node
+SELECT COUNT(*)=1 AS success_on_one_node
+    FROM run_command_on_workers(
+        $$DROP SUBSCRIPTION citus_shard_move_subscription_10_15$$)
+    WHERE success AND result = 'DROP SUBSCRIPTION';
```

Comment:
from experience any query that only returns true or false is hard to debug if it fails for some reason later. The following seems better, at least we'd know if it succeeded on 2 or on 0 nodes.
```suggestion
SELECT *
    FROM run_command_on_workers(
        $$DROP SUBSCRIPTION citus_shard_move_subscription_10_15$$)
    WHERE success AND result = 'DROP SUBSCRIPTION';
```

Reply:
I thought that could cause a flakiness as the node that subscription is created on might change. That's why I have added the count = 1 part, so we can know that the query has succeeded on exactly one node, no matter what the node port is.

How about having just the count part, without =1 check? So that we'd still know:
> at least we'd know if it succeeded on 2 or on 0 nodes.